app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
```

Comment:
about this line, many people will disagree with my decision, since proper way in python is write this code like
```python
if "vaccine" not in visitor_dict
```
But I like that style more, because work with serializers shows, that sometimes you need to check if someone passed there a nullable value or if it's legacy code, which can't check modern syntax.
Anyway, try to investigate a little more about this, to ensure you're getting desirable result.